src/core/Dropdown/Dropdown/Dropdown.tsx

```diff
@@ -76,15 +69,10 @@ export interface DropdownProps {
   visualPlaceholder?: ReactNode;
   /** Show the visual placeholder instead of selected value and act as an action menu */
   alwaysShowVisualPlaceholder?: boolean;
-  /** Custom props for label container */
-  labelProps?: DropdownLabelProps;
-  /** Custom props for Label text element */
-  labelTextProps?: ParagraphProps;
-  /** Label displaymode -
-   * top: show above, hidden: use only for screenreader
-   * @default top
-   */
-  labelMode?: DropdownLabel;
+  /** Label display mode */
+  labelMode?: LabelMode;
+  /** Text to mark a field optional. Will be wrapped in parentheses and shown after labelText. */
+  optionalText?: string;
```

Comment:
In design, **Dropdown** also has the status and status text.

We may consider to add them in this PR or soon, in separate PR.

Reply:
These proposals make sense since we are introducing breaking changes. Maybe we should introduce a quick fix to master with the style fixes required for focus box-sizing? This way we could finalize the components level removal and bring all breaking changes to components at once. Or at least we should try to avoid introducing breaking changes to same components too often.

Reply:
As agreed, this PR will not be included in 3.0 release, so handling these in separate PR's is completely fine. Following tickets should however be created before resolving these comments.

Reply:
Ticket created for adding status and status text.